core/consensus/src/trace.rs

```diff
@@ -18,9 +18,21 @@ impl MetricTracer {
     }
 }
 
+fn err() -> ProtocolError {
+    ConsensusError::Other("tracing: failed when parse point".to_string()).into()
```

Comment:
why not implement `Serde::Error` in ConsensusError?

Reply:
The TracePoint only serialize but not deserialize now, this means that there will be no IO during the processs. So I think error does not occur often